app/jobs/resolution_proofing_job.rb

```diff
@@ -13,7 +13,7 @@ class ResolutionProofingJob < ApplicationJob
   )
 
   def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:, user_id: nil, threatmetrix_session_id: nil)
+              dob_year_only:, user_id: nil, threatmetrix_session_id: nil, uuid_prefix: nil, request_ip: nil)
     timer = JobHelpers::Timer.new
```

Comment:
now that we're adding new arguments, we need to make sure they get set by whatever calls this, so we need to update `idv/agent.rb`